### PR TITLE
Stage 2 changes for RFC 0008 - threat indicator fields

### PR DESCRIPTION
**Pull request by ebeahan**
Apply stage 2 beta changes proposed in the threat indicator fields RFC [0008](https://github.com/elastic/ecs/blob/master/rfcs/text/0008-threat-intel.md): #1293 

[Docs preview of these changes](https://ecs_1478.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-threat.html)

### CODE REVIEW

File: experimental/generated/beats/fields.ecs.yml [hunk 1]
```diff
@@ -9766,32 +9766,19 @@
       level: extended
       type: keyword
       ignore_above: 1024
-      description: "Identifies the confidence rating assigned by the provider using\
-        \ STIX confidence scales.\nExpected values:\n  * Not Specified, None, Low,\
-        \ Medium, High\n  * 0-10\n  * Admirality Scale (1-6)\n  * DNI Scale (5-95)\n\
+      description: "Identifies\_the\_confidence\_rating\_assigned\_by\_the\_provider\_\
```
Comment (djptek) [inline experimental/generated/beats/fields.ecs.yml:9769]: Not sure about the Underscores here?

File: experimental/generated/beats/fields.ecs.yml [hunk 2]
```diff
@@ -9892,10 +9848,10 @@
       type: keyword
       ignore_above: 1024
       description: "Type of indicator as represented by Cyber Observable in STIX 2.0.\n\
-        Expected values\n  * autonomous-system\n  * artifact\n  * directory\n  * domain-name\n\
-        \  * email-addr\n  * file\n  * ipv4-addr\n  * ipv6-addr\n  * mac-addr\n  *\
-        \ mutex\n  * process\n  * software\n  * url\n  * user-account\n  * windows-registry-key\n\
-        \  * x-509-certificate"
+        Recommended values:\n  * autonomous-system\n  * artifact\n  * directory\n\
```
Comment (djptek) [inline experimental/generated/beats/fields.ecs.yml:9851]: Some (but not all) Newlines appear to have been saved as "\n" is this intentional?

File: experimental/generated/ecs/ecs_flat.yml [hunk 3]
```diff
@@ -15226,11 +15226,12 @@ threat.group.reference:
   short: Reference URL of the group.
   type: keyword
 threat.indicator.confidence:
+  beta: This field is beta and subject to change.
   dashed_name: threat-indicator-confidence
-  description: "Identifies the confidence rating assigned by the provider using STIX\
-    \ confidence scales.\nExpected values:\n  * Not Specified, None, Low, Medium,\
-    \ High\n  * 0-10\n  * Admirality Scale (1-6)\n  * DNI Scale (5-95)\n  * WEP Scale\
-    \ (Impossible - Certain)"
+  description: "Identifies\_the\_confidence\_rating\_assigned\_by\_the\_provider\_\
```
Comment (djptek) [inline experimental/generated/ecs/ecs_flat.yml:15231]: Underscores in the description - generated by code from previous comment?

**Review by djptek — COMMENTED** [review]
Can you confirm whether the underscores and extra "\n" characters are intentional? Otherwise looks OK though feedback from a domain SME would also be helpful

File: experimental/generated/beats/fields.ecs.yml (hunk 1 above)
Comment (ebeahan) [reply experimental/generated/beats/fields.ecs.yml]: 👀 Thank you, I didn't notice!

It must have been an artifact left over from how I copied/pasted and how my editor saved the whitespaces. After I edited the line to add the space characters using the space bar, the white spaces now render as expected.

File: experimental/generated/beats/fields.ecs.yml (hunk 2 above)
Comment (ebeahan) [reply experimental/generated/beats/fields.ecs.yml:9851]: I believe the YAML loading/dumping uses an 80 character line length, but then we have our own formatting introducing newlines to get the field description to render nicely in the docs:

https://ecs_1471.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-threat.html#field-threat-indicator-confidence

**Review by peasead — APPROVED** [review]
LGTM

File: experimental/generated/beats/fields.ecs.yml (hunk 2 above)
Comment (peasead) [reply experimental/generated/beats/fields.ecs.yml:9858]: Thanks for the info, Eric.

I'll resolve this conversation, but we can reopen it if needed.

**Review by djptek — APPROVED** [review]
LGTM

**Pull request merged**